core/domain/feedback_domain_test.py

```diff
@@ -161,3 +162,58 @@ def test_to_dict(self) -> None:
         self.assertDictEqual(
             observed_feedback_message_reference.to_dict(),
             expected_feedback_message_reference)
+
+
+class FullyQualifiedMessageIdentifierDomainUnitTests(
+    test_utils.GenericTestBase):
+
+    def setUp(self) -> None:
+        super(FullyQualifiedMessageIdentifierDomainUnitTests, self).setUp()
+        self.exp_id = 'exp'
+        self.message_id = 1
+        self.thread_id = 'exp.thread'
+        self.Message_Identifier = (
+            feedback_domain.FullyQualifiedMessageIdentifier(
+                self.thread_id, self.message_id
+            ))
+
+    def test_initialize(self) -> None:
+        self.assertEqual(
+            self.Message_Identifier.message_id,
+            self.message_id
+        )
+        self.assertEqual(
+            self.Message_Identifier.thread_id,
+            self.thread_id
+        )
+
+
+class FeedbackThreadSummaryDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self) -> None:
+        super(FeedbackThreadSummaryDomainUnitTests, self).setUp()
+        self.exp_id = 'exp'
+        self.message_id = 'message'
+        self.thread_id = 'exp.thread'
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL) # type: ignore[no-untyped-call]
+        self.save_new_valid_exploration(self.exp_id, self.owner_id) # type: ignore[no-untyped-call]
+        self.thread_id_1 = (feedback_services.create_thread( # type: ignore[no-untyped-call]
+            'exploration', self.exp_id, self.owner_id,
+            'A subject', 'Random text'
+        ))
+        self.thread_id_2 = (feedback_services.create_thread( # type: ignore[no-untyped-call]
+            'exploration', self.exp_id, self.owner_id,
+            'Another subject', 'Another Random text'
+        ))
+
+    def test_to_dict(self) -> None:
+        thread_ids = [self.thread_id_1, self.thread_id_2]
+        thread_summeries = (feedback_services.get_exp_thread_summaries( # type: ignore[no-untyped-call]
+            self.owner_id,
+            thread_ids))[0]
+        for i in thread_summeries:
+            self.assertIn(
+                (i.to_dict())['thread_id'],
```

Comment:
Can you also check that the rest of the thread is correct? This might be easiest to test if you move the thread creation code from `setUp()` to `test_to_dict()`

Reply:
Thanks for pointing it out. I have changed the test.